### ISSUE TITLE
Going through anti spam options

### ISSUE THREAD

**Issue by paaljoachim** [fluentform/fluentform#451]
I am going through various Anti spam options. Some brainstorming.

I see that Fluent Forms includes:
reCaptcha, hCaptcha and Turnstile. Great!

I have earlier found in other form plugins the following anti spam options:
Math captcha manually made (Contact Form 7. Example here: https://verden.no/reisebyra-bergen-os-kontakt/ )
Honeypot (I do not know if this is automatically included or if a user needs to do something to turn it on in Fluent Forms.)
Akismet (Again it is not clear if the Forms will use Akismet or not.)
Adding a Hidden field which a bot will fill out but not a human. https://stackoverflow.com/questions/36227376/better-honeypot-implementation-form-anti-spam sounds like a good idea. Adding a hidden field and having the option if it is filled out then it can be marked as spam.

I came across this Fluent Forms blog post:
https://fluentforms.com/form-security/

Settings -> Scheduling & Restrictions - Empty Submission Blocking: Deny empty submission. Is OFF by default. This should be ON by default as one should not be able to submit an empty form.

Bottom line is that I miss having a math field to prevent spam, using Akismet, and a hidden field to prevent spam. Perhaps some of these options are already in Fluent Forms. If these are present then it should be made more obvious. As I could not locate these in the fields area.
